src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,87 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int RESIZE_VALUE = 2;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+    }
+
     @Override
     public void put(K key, V value) {
-
+        if (size == table.length * DEFAULT_LOAD_FACTOR) {
```

Comment:
better move this to the method
```suggestion
        if (size > getThreshold()) {
```